### ISSUE TITLE
Print.SetOptions operation failed. Unable to save the settings. 

### ISSUE THREAD

**Issue by Crusader36** [erfajo/OrchidForDynamo#256]
### Operating system
Windows 10

### Revit and Dynamo version
Revit: 2023.1
Dynamo: 2.16.2
Orchid: 2.16.6.6324

### What did you do? 
Creating print settings, setting options, printing and deleting the settings.

### What did you expect to see?
I expect creation of print setting followed by set options and pdf export and deletion of those settings.

### What did you see instead?
I am getting an error at set option node that "PrintOptions failed. Save of the print setup was unsuccessful". The print setting is getting created but unable to execute print and no deletion as seen below.  

This is the same workflow as shown in your sample files while dealing with print nodes. I am not sure if you can recreate it at your as it is working in some of the central model and failing at some. 

![image](https://github.com/user-attachments/assets/7327da3a-c14e-4e7a-865e-7c4af76e63d7)

Thanks

**Comment by erfajo** [comment]
Thanks for reporting, I will look into this asap :-)

**Comment by erfajo** [comment]
I have now tested in my environment, and can't find any error. Could it be the case that what you are trying to do is in a not saved document? If not, can you send me the graph file? You can add .txt to the extension and then upload it.

**Comment by Crusader36** [comment]
No, both the Revit and dynamo files are saved before running script. I typically close and open .dyn file for every run. As I said in some other central models it is working fine, I could not figure out the issue.  Please find the graph below. 

[batch print orchid TEMP.txt](https://github.com/user-attachments/files/17025394/batch.print.orchid.TEMP.txt)

**Comment by erfajo** [comment]
Sorry but I got no errors at all, however, I can see that you have an older version of Orchid, try to update.
Besides, I have made some few changes in your GRAPH (not script, that i bad slang) you maybe can use.
[batch.print.orchid_erfajo.dyn.txt](https://github.com/user-attachments/files/17026602/batch.print.orchid_erfajo.dyn.txt)

**Comment by Crusader36** [comment]
I updated Orchid. It is not working in some files.  I will do a clean reinstall of Revit and see if helps.

**Comment by Crusader36** [comment]
I have cleaned files and reinstalled Revit only to find that the issue remain. As a workaround I have shifted to PDFexport nodes, and it worked fine. However, I had to add additional transaction node to make it work. Thanks for your time.


![2024-09-19 15_29_04-Dynamo](https://github.com/user-attachments/assets/a74408da-4438-4db1-9f20-ff41bd38da97)

**Comment by erfajo** [comment]
Well since I cant replicate your errors I dont know how to help.
However, something is really going bad in your installation, since that function dont need a transaction.
I will close this issue.

**Comment by Crusader36** [comment]
After finding difficulty in some of the projects, I reached out of Revit technical support but they said it is not in their scope. I took help from dynamo forum, chatGPT and developed python script to create setting with set options in one node. It is now working in all my projects. Sharing my graph here just in case anyone else faces the same problem. 

[batch print orchid TEMP.txt](https://github.com/user-attachments/files/17400756/batch.print.orchid.TEMP.txt)
